### ISSUE TITLE
Why use batch size 2 for training?

### ISSUE THREAD

**Issue by qinjian623** [sel118/LaneAF#13]
Hi, this is a hp setting confusing me.

Empirically, batch size greater than 8 is good for training.
 
So why use bs=2? Any special consideration for this hyper-parameter setting? 

Or to simulate instance norm without change backbone's batch norm?
